CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## X.X.X (YYYY-MM-DD)
 
+-   Fix an issue that occurs when writing attributes named `domains`, `domainsLevel2` and `documentation`.
```

Comment:
What was happening exactly (not for the RNs, but for the review xD)? And what does “writting attributes” mean, here? “Setting”? But weren’t people setting `documentation` fairly often already? I’m lost. :frowning_woman:

Reply:
Because the extension parses the file even when we are writing code − which makes the extension parse incomplete code − some useless errors were caught and VSCode was yelling that our extension warn about an error.

Reply:
AH.

Reply:
So I can just open existing stuff and mess around with the docs, haha.

Reply:
```suggestion
-   Fix a parsing issue that occurs when editing the value of attributes named `domains`, `domainsLevel2` and `documentation`.
```

Reply:
Yes.
You can try with something like:
```
Object MY_OBJ
--> documentation // "Uncomment me!"
;
```